wasm/jit/jit_amd64.go

```diff
@@ -4115,11 +4225,21 @@ func (c *amd64Compiler) setJITStatus(status jitCallStatusCode) {
 	c.addInstruction(prog)
 }
 
-func (c *amd64Compiler) callHostFunctionFromConstIndex(index int64) error {
-	// Set the jit status as jitCallStatusCodeCallHostFunction
-	c.setJITStatus(jitCallStatusCodeCallHostFunction)
-	// Set the function index.
-	c.setFunctionCallIndexFromConst(index)
+// makeFunctionCallFromConsts makes a function call against the function whose address equals addr parameter.
+// jitStatus is set before making call, and it should either jitCallStatusCodeCallBuiltInFunction or
+// jitCallStatusCodeCallFunction.
+func (c *amd64Compiler) makeFunctionCallFromConsts(jitStatus jitCallStatusCode, addr wasm.FunctionAddress) error {
```

Comment:
const is correct but address is more user level, right?

```suggestion
func (c *amd64Compiler) compileFunctionCallFromAddress(jitStatus jitCallStatusCode, addr wasm.FunctionAddress) error {
```